mobile/src/components/course/course.tsx

```diff
@@ -51,14 +78,25 @@ const Course: FC = () => {
   return (
     <ScrollView>
       <View style={styles.container}>
-        <Text style={styles.h1}>{dataCourse?.title}</Text>
+        <Text style={styles.h1}>{(course as CourseGetResponseDto).title}</Text>
+        <View style={styles.currentCategory}>
+          <Category
+            keyName={currentCategory?.key ?? 'unknown'}
+            name={currentCategory?.name ?? 'Unknown'}
+            isActive={false}
+          />
+        </View>
+
         <Image
           style={styles.image}
           source={{ uri: course?.imageUrl ?? getImageUri(defaultCourseImage) }}
         />
         <Text style={styles.h2}>About this course</Text>
         {Boolean(course?.description) && (
-          <Content html={dataCourse?.description} width={width} />
+          <Content
+            html={(course as CourseGetResponseDto).description}
```

Comment:
We have check about this above
```suggestion
            html={course.description}
```

Reply:
You mean add `const dataCourse = course as CourseGetResponseDto;` and then I can use  `html={course.description}` ?

Reply:
```
{courseData.description && (
  <Content
    html={courseData.description}
```